README.md

```diff
@@ -1,109 +1,15 @@
 ![cf](https://i.imgur.com/7v5ASc8.png) Lab 07: Vanilla HTTP Server
```

Comment:
your readme needs to tell users how to operate your application.  eg-how does one make the cow say something?